cmd/proxy-server/main.go

```diff
@@ -587,8 +587,9 @@ func refreshAdminTokenHandler(log *logrus.Entry) http.Handler {
 		var input token.AdminToken
 		err := json.NewDecoder(r.Body).Decode(&input)
 		if err != nil {
-			log.WithError(err).Error("decoding admin token pair")
-			http.Error(w, "decoding admin token pair", http.StatusInternalServerError)
+			if err := web.JSONErrorResponse(w, http.StatusInternalServerError, fmt.Errorf("decoding admin token pair")); err != nil {
```

Comment:
The error can be included here:

fmt.Errorf("decoding admin token pair: %v", err)

Reply:
fixed